packages/elements-react/package.json

```diff
@@ -59,18 +59,14 @@
     "eslint-plugin-prefer-arrow": "^1.2.3",
     "eslint-plugin-react": "^7.22.0",
     "np": "^3.1.0",
-    "react": "^16.9.0",
-    "react-dom": "^16.9.0",
-    "react-router": "^4.3.1",
-    "react-router-dom": "^4.3.1",
+    "react": "^17.0.2",
```

Comment:
There's a breaking change in here we need to document: The React version changes from 16 -> 17 which breaks a few things for the consuming applications.